### ISSUE TITLE
Update to newer AndroidX packages with net6 support

### ISSUE THREAD

**Issue by Redth** [dotnet/maui#1904]
There are now AndroidX bindings on NuGet with net6.0-android30.0 assets.  We need to update MAUI to use them.  This may help reduce the legacy assemblies brought in as well.

**Comment by jonathanpeppers** [comment]
@jpobst were you going to send a PR?

I think this would be updating all the version numbers here:

https://github.com/dotnet/maui/blob/main/eng/AndroidX.targets

And remove `$(_AndroidXVersion)`, we don't need it anymore.

**Comment by jpobst** [comment]
I think I'll leave it to the MAUI team.  I suspect they will want to test them first.

**Comment by jonathanpeppers** [comment]
Yeah, I can send a PR that get's it to build at least. We'll probably need someone to manually test, thanks.

**Comment by jonathanpeppers** [comment]
Current blocker is:
```
error NU1101: Unable to find package Xamarin.AndroidX.Tracing.Tracing. No packages exist with this id in source(s): dotnet-eng, dotnet-public, dotnet6, xamarin [C:\src\maui\Microsoft.Maui-net6.sln]
```

We'll need to get this package listed on `dotnet-public`. It must be a new dependency?
